src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,83 @@
+.product-card {
+  box-sizing: border-box;
```

Comment:
move to `*` selector in normalize, `box-sizing` should be the same for all elements